tests/e2e/cucumber/steps/ui/a11.y.ts

```diff
@@ -0,0 +1,11 @@
+import { Then } from '@cucumber/cucumber'
+import { World } from '../../environment'
+import { checkAccessibility } from '../../../support/utils/accessibility'
+
+Then(
+  '{string} check accessibility of the {string} page',
+  async function (this: World, stepUser: string, place: string): Promise<void> {
+    const { page } = this.actorsEnvironment.getActor({ key: stepUser })
+    await checkAccessibility(page, `personal space page`, place)
```

Comment:
`personal space page` will be only true for some tests.
Can we get enough context e.g. from the URL or maybe we can put it into the step definition
`'{string} checks the accessibility of the DOM element {string} on the {string}`
so in the feature file it will be: `When "Alice" checks the accessibility of the DOM element ".files-view-wrapper" on the "personal space page"`